lib/gems/pending/VMwareWebService/VimService.rb

```diff
@@ -1249,16 +1253,32 @@ def marshalObj(node, obj)
     end
   end
 
-  def parse_response(response, rType)
-    doc  = response.document
-    raise "Response <#{response.inspect}> has no XML document" if doc.nil?
+  def handle_memory_and_gc(response)
```

Comment:
It is under https://github.com/agrare/manageiq-gems-pending/blob/b69d853254d43b4ccd771df9568c1e1fe582a980/lib/gems/pending/VMwareWebService/VimService.rb#L1218, https://github.com/agrare/manageiq-gems-pending/blob/b69d853254d43b4ccd771df9568c1e1fe582a980/lib/gems/pending/VMwareWebService/VimService.rb#L1256